src/com/google/enterprise/adaptor/fs/FsAdaptor.java

```diff
@@ -423,6 +433,8 @@ public void initConfig(Config config) {
     config.addKey(CONFIG_MONITOR_UPDATES, "true");
     config.addKey(CONFIG_STATUS_UPDATE_INTERVAL_MINS, "15");
     config.addKey(CONFIG_SEARCH_RESULTS_GO_TO_REPO, "true");
+    // TODO (bmj): should this default to true?
```

Comment:
I think true is more desirable, but the risk is whether this code works with most SMB/CIFS appliances, which are mostly not tested.